packages/controller-utils/src/util.ts

```diff
@@ -426,20 +444,24 @@ export function normalizeEnsName(ensName: string): string | null {
  * @returns Promise resolving the request.
  */
 export function query(
-  ethQuery: any,
+  ethQuery: EthQueryLike,
   method: string,
   args: any[] = [],
 ): Promise<any> {
   return new Promise((resolve, reject) => {
-    const cb = (error: Error, result: any) => {
+    const cb = (error: unknown, result: unknown) => {
       if (error) {
         reject(error);
         return;
       }
       resolve(result);
     };
 
-    if (typeof ethQuery[method] === 'function') {
+    if (
+      hasProperty(ethQuery, method) &&
```

Comment:
This seems to have introduced a bug; the methods in question are on the `ethQuery` prototype, so this is returning `false` when this condition used to resolve to `true`

Reply:
Right, that makes sense.